Code of conduct updates dec2020 (#22)
EquitableLettersSP/ELSP: CODE_OF_CONDUCT.md

# ELSP Code of Conduct

We at ELSP have chosen to adopt and follow the standards outlined in the
Contributor Covenant Code of Conduct written for the development and use for
Open Source Projects, at
[Version 2.0](https://www.contributor-covenant.org/version/2/0/code_of_conduct/).
This is version 1.0 of our Code of Conduct (ADD DATE AFTER ADDING LINKS).

## Our Pledge

> We as editors, reviewers, and writers pledge to follow in the name of our

Reviewer: It says "pledge to follow", should this be "pledge to follow this code of conduct"

Author: ```suggestion
We as editors, reviewers, and writers pledge to follow this code of conduct in the name of our
```

Author: I'm using the little (+/-) button to make suggestions that can be committed directly from here.

- Timely responses from letter reviewer(s) and submitter(s).
- Disclosing submission of a letter/recommendation to ELSP to proper overseeing
  bodies following the standards of a person's employer.
- Focusing on what is best for the person(s) described in letter/recommendation.

We offer the following additional example(s) of behavior that are unacceptable:

- The use of sexualized language or imagry, and sexual attention or advances of
  any kind.
- Trolling, insulting or derogatory comments, and personal or political attacks.
- Public or private harassment.
- Revealing information reviewed in confidence.
- Publishing others' private information, such as a physical or email address,
  without their explicit permission.
- Violation of the confidentiality of the review process without the consent of
  all of the involved parties.
- Not adhering to the guidelines for submitters and reviews (LINK HERE).

## Enforcement Responsibilities

Original creators and the current editorial board volunteers at ELSP are
responsible for elucidating and enforcing standards of acceptable and
unacceptable behavior.

In the case of unacceptable behavior, ELSP, has the right to refuse to review

> and or provide feedback to contributors whose actions are not aligned with this

Reviewer: Should it be "and/or" instead of "and or"?

Author: ```suggestion
and/or provide feedback to contributors whose actions are not aligned with this
```

and the Covenant Code of Conduct.

## Scope

This Code of Conduct applies to all community spaces, and also applies when an
individual is officially representing the community in public spaces.  Examples
of representing our community include, but are not limited to, using an official
e-mail address and acting as a representative at a conference.  Examples of
community spaces include, but are not limited to, official e-mail
correspondence, the GitHub repository, and officially sponsored offline events.

## Enforcement

Instances of abusive, harassing, or otherwise unacceptable behaviour may be
reported to the editorial board at equitable.space.letters@gmail.com.  All
complaints will be reviewed and investigated promptly and fairly.

All community leaders are obligated to respect the privacy and security of the
reporter of any incident.

### Enforcement Guidelines

Here, ELSP will follow the Covenant Code of Conduct in determining the

> consequences any action(s) deemed to be a violation of this Code of Conduct:

Reviewer: Should be "consequences of any action(s)"

Author: ```suggestion
consequences of any action(s) deemed to be a violation of this Code of Conduct:
```

2.) Warning

Community Impact: A violation through a single incident or series of actions.

Consequence: A warning with consequences for continued behavior. No interaction
with the people involved, including unsolicited interaction with those enforcing
the Code of Conduct, for a specified period of time. This includes avoiding
interactions in community spaces as well as external channels like social media.
Violating these terms may lead to a temporary or permanent ban.

3.) Temporary Ban

Community Impact: A serious violation of community standards, including
sustained inappropriate behaviour.

Consequence: Barred from all community spaces (including acting as a reviewer or
submitting a letter for review) for a specified period of time.  No public or
private interaction with involved parties, including unsolicited interaction
with those enforcing the Code of Conduct, should be initiated during this
period.  Violating these terms may lead to a permanent ban.  If the temporary
ban occurs in response to actions against an individual, the editorial board
will offer to back the offended party should they wish to take further action,

> including notifying the offenders' institution or professional organisations.

Reviewer: Replace "organisations" with "organizations"

Author: ```suggestion
including notifying the offenders' institution or professional organizations.
```

4.) Permanent Ban

Community Impact: Demonstrating a pattern of violation of community standards,
including sustained inappropriate behaviour, harassment of an individual, or
aggression toward or disparagement of classes of individuals.

Consequence: A permanent ban from the community.  May not act as a reviewer or
editor, may not submit letters for review, may not participate in online or
offline spaces.  The editorial board will contact the offenders' institution

> and relevant professional organisations if the behaviour also violates the

Reviewer: "Replace "organisations" with "organizations"

Author: ```suggestion
and relevant professional organizations if the behavior also violates the
```

Code of Conduct of those communities.  If the permanent ban occurs in response
to actions against an individual, the editorial board will offer to back the
offended party should they wish to take further action.

## Attribution

> This code of Conduct is adapted from the

Reviewer: Replace "code" with "Code"

Author: ```suggestion
This Code of Conduct is adapted from the
```
